ddtrace/internal/utils/attrdict.py

```diff
@@ -1,10 +1,9 @@
 from typing import Any
+from typing import Callable
 
 
-class AttrDict(dict):
-    """
-    dict implementation that allows for item attribute access
-
+class AttrDict(object):
```

Comment:
do we want to inherit/implement `collections.abc.Mapping`?
that might be useful for typing